src/handlers/covalent.ts

```diff
@@ -0,0 +1,41 @@
+import { captureException } from '@sentry/react-native';
+import {
+  // @ts-ignore
+  COVALENT_ANDROID_API_KEY,
+  // @ts-ignore
+  COVALENT_IOS_API_KEY,
+} from 'react-native-dotenv';
+import { rainbowFetch } from '../rainbow-fetch';
+import { EthereumAddress } from '@rainbow-me/entities';
+import Logger from 'logger';
+export const getAssetsFromCovalent = async (
+  chainId: Number,
+  address: EthereumAddress,
```

Comment:
can we stick to `accountAddress` naming convention

Reply:
same thing in the files where this is called

Reply:
also while we're at it, can you insert a space between lines 10 and 11

Reply:
Sure. we're moving it to the backend anyways so I don't care.